draft-ietf-quic-applicability.md

```diff
@@ -484,29 +491,29 @@ identification.
 
 # Connection Migration
 
-QUIC supports connection migration by the client. If a lower-layer address
+QUIC supports connection migration by the client. If an IP address
 changes, a QUIC endpoint can still associate packets with an existing
-connection using the Destination connection ID field (see also {{connid}}) in
-the QUIC header, unless a zero-length value is used. This supports cases where
-address information changes, such as NAT rebinding, intentional change of the
-local interface, or based on an indication in the handshake of the server for a
-preferred address to be used.
+transport connection using the Destination Connection ID field
+(see also {{connid}}) in the QUIC header, unless a zero-length value is used.
+This supports cases where address information changes, such as NAT rebinding,
+intentional change of the local interface, or based on an indication in the
+handshake of the server for a preferred address to be used.
 
 Use of a non-empty connection ID for the server is strongly recommended if any
 clients are behind a NAT or could be. A non-empty connection ID is also
 strongly recommended when migration is supported.
 
 Currently QUIC only supports failover cases. Only one "path" can be used at a
 time, and only when the new path is validated all traffic can be switched over
-to that new path. Path validation means that the other endpoint in required to
+to that new path. Path validation means that the remote endpoint is required to
 validate the new path before use in order to avoid address spoofing attacks.
 Path validation takes at least one RTT and congestion control will also be reset
-on path migration. Therefore migration usually has a performance impact.
+after path migration. Therefore migration usually has a performance impact.
 
-Probing packets, which cannot carry application data, can be sent on multiple
-paths at once. Probing packets can be used to perform address validation,
-measure path characteristics as input for the switching decision, or prime the
-congestion controller in preparation for switching to the new path.
+QUIC probing packets, which cannot carry application data, can be sent on
+multiple paths at once. Probing packets can be used to perform address
+validation, measure path characteristics as input for the switching decision,
+or prime the congestion controller in preparation for switching to the new path.
```

Comment:
/prime/ seems like a topic that has some semantics that needs CC discussion, and I don't know where I see this in RFCs. This is discussed issue #197.

Reply:
Although I see PR #122 used different language and avoided having to discuss this:-).

Reply:
This change was just changing the line alignment not the words. Let's keep discussion in #197 for this topic.